docs/en/Configuring-Alluxio-with-Azure-Blob-Store.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: global
+title: Configuring Alluxio with Azure Blob Store
+nickname: Alluxio with Azure Blob Store
+group: Under Store
+priority: 0
+---
+
+* Table of Contents
+{:toc}
+
+This guide describes how to configure Alluxio with [Azure Blob Store](https://azure.microsoft.com/en-in/services/storage/blobs/) as the under storage system.
+
+## Initial Setup
+
+To run an Alluxio cluster on a set of machines, you must deploy Alluxio binaries to each of these machines. You can either [compile the binaries from Alluxio source code](http://alluxio.org/documentation/master/Building-Alluxio-Master-Branch.html), or [download the precompiled binaries directly](http://alluxio.org/documentation/master/Running-Alluxio-Locally.html).
```

Comment:
I think the link url should only be: `Building-Alluxio-Master-Branch.html` and `Running-Alluxio-Locally.html`.